R/systemfit-tidiers.R

```diff
@@ -0,0 +1,52 @@
+#' @templateVar class systemfit
+#' @template title_desc_tidy
+#' 
+#' @param x A `systemfit` object produced by a call to [systemfit::systemfit()].
+#' @param conf.int Logical indicating whether or not to include a confidence interval in the tidied output. Defaults to `FALSE`.
```

Comment:
Can you add an additional argument `conf.level` defaulting to 0.95 to allow the use to set the confidence level? Then you can simultaneously document both arguments using the `@template param_confint` to achieve consistent documentation!

Reply:
This was actually a deliberate omission. Totally happy to add this, but didn't originally because `conf.level` doesn't change the CIs in the method that they've created for it in the original systemfit package, so it seemed a bit disingenuous to tell users they could input this. LMK your preference

Reply:
Would you add this? Unless it totally isn't possible, in which case you should set `strict = FALSE` in the `check_arguments()`  test and document that this is the case.

Reply:
Added.